File: SD3/java examples/Tutorial_1_wbBall/Ball.java
import javax.swing.JOptionPane;
public class Ball {
	//properties
	protected String colour;
	protected String Name; 
	//constructors
	public Ball(){}
	
	public Ball(String colour, String name){
		setColour(colour);
		setName(name);
	}
	
	//gets and sets
	public void setColour(String Colour){this.colour = Colour; }
	public String getColour() {return this.colour; }
	public void setName(String Name) {this.Name = Name;}
	public String getName() {return this.Name;}
	
}

File: SD3/Tutorial 1_Q1/Tutorial 1_Q1/Ball.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Tutorial_1_Q1
{
    class Ball
    {
        //properties
        protected string Name;

        //Constructors
        public Ball() { }
        public Ball(String Name) { this.Name = Name; }

        //gets and sets
        public String getName() { return Name; }
        public void setName(string Name) { this.Name = Name; }

        //methods
    }
}
